load.php

```diff
@@ -108,7 +108,23 @@ function perflab_get_module_settings() {
 	// Even though a default value is registered for this setting, the default must be explicitly
 	// passed here, to support scenarios where this function is called before the 'init' action,
 	// for example when loading the active modules.
-	return (array) get_option( PERFLAB_MODULES_SETTING, perflab_get_modules_setting_default() );
+	$perflab_get_module_settings = (array) get_option( PERFLAB_MODULES_SETTING, perflab_get_modules_setting_default() );
+
+	$legacy_module_slug = array(
+		'site-health/audit-autoloaded-options' => 'database/audit-autoloaded-options',
+		'site-health/audit-enqueued-assets'    => 'css-and-js/audit-enqueued-assets',
```

Comment:
See above. Please also make sure to then rename the directories of the code for that module.